ess/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Generic SAL configuration
+FROM centos:centos7
+RUN yum update -y \
+  && yum install -y git gcc gcc-c++ make openssl-devel bzip2-devel libffi-devel ncurses-libs xterm xorg-x11-fonts-misc java-1.8.0-openjdk-devel maven swig tk-devel vim bash-completion bash-completion-extras wget libgphoto2-devel \
+  && yum clean all
+RUN wget https://www.python.org/ftp/python/3.7.8/Python-3.7.8.tgz \
+  && tar xzf Python-3.7.8.tgz \
+  && cd Python-3.7.8 \
+  && ./configure --enable-optimizations --enable-shared --with-system-expat --with-ensurepip=yes --with-system-ffi \
+  && make -j4 build_all \
+  && make altinstall
+RUN echo "/usr/local/lib" > /etc/ld.so.conf.d/python3.conf \
+  && ldconfig
+RUN cd /usr/local/bin \
+  && ln -s python3.7 python3 \
+  && ln -s python3 python \
+  && ln -s pip3.7 pip3 \
+  && pip3 install --upgrade pip
+RUN wget https://repo-nexus.lsst.org/nexus/repository/ts_yum/releases/OpenSpliceDDS-6.9.0-8.armv7hl.rpm \
+  && rpm -i OpenSpliceDDS-6.9.0-8.armv7hl.rpm \
+  && export OSPL_HOME=/opt/OpenSpliceDDS/V6.9.0/HDE/armv7l.linux-debug \
+  && export PATH=${OSPL_HOME}/bin:${PATH} \
+  && export LD_LIBRARY_PATH=${OSPL_HOME}/lib \
+  && pip3 install cython \
+  && pip3 install wheel \
+  && cd /tmp \
+  && wget https://files.pythonhosted.org/packages/21/de/edf62206777581aad3f3b7d5f7e1360955e2c1f837061ccd7eb2cf1acaed/ts-dds-6.9.190925.post7.tar.gz \
+  && tar zxf ts-dds-6.9.190925.post7.tar.gz \
+  && cd ts-dds-6.9.190925.post7 \
+  && cp $OSPL_HOME/tools/python/src/dds* . \
+  && python3 setup.py install \
+  && pip3 install ts-salobj ts-idl
+# ESS specific configuration
+RUN pip3 install pyserial RPi.GPIO RPIO pigpio gpiozero
+RUN pip3 install RPi.GPIO RPIO pigpio gpiozero
+RUN adduser -G wheel,dialout ess
+USER ess
+WORKDIR /home/ess
+RUN git clone https://github.com/lsst-ts/ts_ess.git \
+  && git clone https://github.com/lsst-ts/ts_config_ocs.git
+RUN echo 'export TS_CONFIG_OCS_DIR=/home/ess/ts_config_ocs' >> .bashrc \
+ && echo 'export OSPL_HOME=/opt/OpenSpliceDDS/V6.9.0/HDE/armv7l.linux-debug' >> .bashrc \
+ && echo 'export OSPL_URI=file://${OSPL_HOME}/etc/config/ospl.xml' >> .bashrc \
+ && echo 'export PATH=${OSPL_HOME}/bin:${PATH}' >> .bashrc \
+ && echo 'export LD_LIBRARY_PATH=${OSPL_HOME}/lib' >> .bashrc \
+ && echo 'export LSST_DDS_DOMAIN=ess' >> .bashrc \
+ && echo 'export PYTHONPATH=/home/ess/ts_ess/python' >> .bashrc
+ENTRYPOINT /bin/bash
```

Comment:
I don't think you need this `ENTRYPOINT` step.

Reply:
Thanks, I removed them from all files.